### PR TITLE
Fix a typo in scala3 book

### PR DESCRIPTION
**Pull request by berberman**

### CODE REVIEW

**Review by Philippus — APPROVED** [review]

File: _overviews/scala3-book/packaging-imports.md [hunk 1]
```diff
@@ -211,7 +211,7 @@ package foo
 
 class ClassA:
   import scala.util.Random   // inside ClassA
-  def printRandom {
+  def printRandom:
```
Comment (julienrf) [inline _overviews/scala3-book/packaging-imports.md:214]:
```suggestion
  def printRandom(): Unit =
```

**Review by julienrf — CHANGES_REQUESTED** [review]
The change you proposed is incorrect. Please see my suggestion for an alternative.

File: _overviews/scala3-book/packaging-imports.md (hunk 1 above)
Comment (griggt) [reply _overviews/scala3-book/packaging-imports.md:214]: Looks like line 202 needs this change as well.

**Review by julienrf — APPROVED** [review]
Thank you for the fix!

**Pull request merged**